### ISSUE TITLE
coingecko crashed no results since yesterday

### ISSUE THREAD

**Issue by code4crypto** [omeryousaf/sasta-miner#46]
![image](https://user-images.githubusercontent.com/88612475/128812819-f967437f-19eb-483d-8af5-93b07790753a.png)
no results have appeared since alpha kombat on telegram or discord - CG bot seems down

**Comment by code4crypto** [comment]
@omeryousaf

**Comment by omeryousaf** [comment]
Thanks for logging the issue @code4crypto . The last logs i see from coingecko scraping service  seem to be from 9th August at around 9:37pm as you can see below (also, you can see how frequently it rate limits us, each time for 62000 seconds, @Shapokhan plz correct me if i am wrong). There was no crash report but it seems to have gone offline after that. I have restarted the service for now. I believe this is the same issue we have already logged #44 .
```
2|sasta    | 2021-08-09T16:10:54: sent notification to telegram bot
2|sasta    | 2021-08-09T16:11:24: Response Status = 429 Interval = 62000
2|sasta    | 2021-08-09T16:12:46: Response Status = 429 Interval = 62000
2|sasta    | 2021-08-09T16:14:18: Response Status = 429 Interval = 62000
2|sasta    | 2021-08-09T16:15:41: Response Status = 429 Interval = 62000
2|sasta    | 2021-08-09T16:17:16: Response Status = 429 Interval = 62000
2|sasta    | 2021-08-09T16:18:38: Response Status = 429 Interval = 62000
2|sasta    | 2021-08-09T16:20:10: Response Status = 429 Interval = 62000
2|sasta    | 2021-08-09T16:21:33: Response Status = 429 Interval = 62000
2|sasta    | 2021-08-09T16:23:10: Response Status = 429 Interval = 62000
2|sasta    | 2021-08-09T16:24:33: Response Status = 429 Interval = 62000
2|sasta    | 2021-08-09T16:26:00: Response Status = 429 Interval = 62000
2|sasta    | 2021-08-09T16:27:28: Response Status = 429 Interval = 62000
2|sasta    | 2021-08-09T16:28:58: Response Status = 429 Interval = 62000
2|sasta    | 2021-08-09T16:30:26: Response Status = 429 Interval = 62000
2|sasta    | 2021-08-09T16:31:48: Response Status = 429 Interval = 62000
2|sasta    | 2021-08-09T16:33:20: Response Status = 429 Interval = 62000
2|sasta    | 2021-08-09T16:34:47: Response Status = 429 Interval = 62000
2|sasta    | 2021-08-09T16:36:19: Response Status = 429 Interval = 62000
2|sasta    | 2021-08-09T16:37:42: Response Status = 429 Interval = 62000
```

**Comment by code4crypto** [comment]
but once rate limit is removed we should still be able to get new listings as we compare to last published listing, no?

**Comment by omeryousaf** [comment]
No, we don't store data offline, so we lose the intermediate results between a crash and subsequent restart. A restart is like a fresh start in our case.

**Comment by omeryousaf** [comment]
Fixed by #48
